demisto_sdk/commands/lint/linter.py

```diff
@@ -990,13 +990,22 @@ def _docker_image_create(self, docker_base_image: List[Any]) -> Tuple[str, str]:
 
     def _docker_remove_container(self, container_name: str):
         try:
+            logger.info(f"Trying to remove container {container_name}")
             container = self._docker_client.containers.get(container_name)
+            logger.info(f"Found container {container_name=}, {container.id=}")
             container.remove(force=True)
+            logger.info(f"Successfully removed container {container_name}")
         except docker.errors.NotFound:
+            logger.info(f"Didn't remove container {container_name} as it wasn't found")
             pass
         except requests.exceptions.ChunkedEncodingError as err:
             # see: https://github.com/docker/docker-py/issues/2696#issuecomment-721322548
-            if platform.system() != "Darwin" or "Connection broken" not in str(err):
+            system_is_not_darwin = platform.system() != "Darwin"
+            not_connetion_broken = "Connection broken" not in str(err)
+            logger.info(
```

Comment:
either error or warning imo

Reply:
It's not necessarrily an error.
We try to remove the container anyway.
If it's not found - We're good.
But we want to know if it didn't find it, and then we'll get:
`Conflict. The container name "foo" is already in use by container "bar". You have to remove (or rename) that container to be able to reuse that name.`